### DISCUSSION TITLE
Cannot Access my Profile Tab Anymore

### DISCUSSION THREAD

**Discussion by CelineWuest** [community/community#66992] [Mobile] [answered]
Not sure when exactly it started happening, but whenever I try to switch to my profile tab, the app crashes consistently.

**Comment by iskae** [comment, marked as answer]
Hello @DominicWuest 👋 

Thanks a bunch for reaching out ❤️ We are aware of this issue and already fixed it 🎉 

Please check out our latest version and let us know if the issue persists 🙇 

Cheers!
